src/utils/AxiosErrorHandling.ts

```diff
@@ -0,0 +1,11 @@
+import axios, { AxiosError } from 'axios';
+
+export const handlePotentialAxiosError = (error: unknown): AxiosError | Error => {
+  if (axios.isAxiosError(error)) {
+    return error;
+  } else if (error instanceof Error) {
+    return error;
+  } else {
+    return new Error('api call failed');
```

Comment:
Kanskje en enda mer generisk tekst.Vi kan jo ikke garantere at metoden kun brukes for å håndtere api-kall.

Reply:
fikset